src/paper_feeds/services/journal_service.py

```diff
@@ -0,0 +1,45 @@
+from paper_feeds.config import config
+from paper_feeds.db import get_engine
+from paper_feeds.models.journal import Journal, ISSN
+from paper_feeds.services import openalex
+from sqlmodel import Session, select
+
+
+def get_journals_homepage(journals: list[Journal]):
+    # extract one issn of each journal newly added to DB
+    issns_to_query = []
+    for journal in journals:
+        if journal.issn:
+            issns_to_query += [journal.issn[0].value]
```

Comment:
note to self, we should add a way of doing 'preferred issn' and getting the web ISSN if available, i have a hunch that the assumption that metadata will be equal across ISSNs (so we can treat them as arbitrary) will prove incorrect, but for now this is fine!